tests/coverage_llvm-cov/test_llvm-cov.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+DIR_BUILD=$PWD
+DIR_SRC=$(cd $(dirname "$0")/../..; pwd -P)
+DIR_GTEST_BUILD="$DIR_BUILD/gtest"
+DIR_GTEST_INSTALL="$DIR_GTEST_BUILD/googletest/install"
+GTEST_SETUP="$DIR_SRC/ci/travis/setup_gtest.sh"
+DATE_TIME=$(date +"%y-%m-%d-%H%M%S")
+DIR_REPORT="$DIR_BUILD/llvm-cov/report_coverage_${DATE_TIME}"
+REPORT_COVERAGE="$DIR_REPORT/index.html"
+REPORT_CTEST="$DIR_REPORT/ctest.log"
+COVERAGE_COLLECT="${DIR_BUILD}/coverage.info"
+COVERAGE_FILTERED="${DIR_BUILD}/coverage_filtered.info"
+FILE_PROFDATA="code.profdata"
+DUMP_PROFRAW="dump-%p.profraw"
```

Comment:
variables should have meaningful names, which help understanding what the script does. "DUMP_PROFRAW" is meaningless. RAW_PROFILER_OUTPUT ?
Consider getting rid of that one anyway.

Reply:
Agree. Believe it or not, I was thinking about this change.
https://www.thesaurus.com/browse/filtered